src/main/kotlin/no/nav/familie/ef/sak/api/ekstern/PerioderOvergangsstønadController.kt

```diff
@@ -14,21 +14,15 @@ import org.springframework.web.bind.annotation.RestController
 
 
 @RestController
-@RequestMapping(path = ["/api/ekstern/periode/overgangsstonad"],
+@RequestMapping(path = ["/api/ekstern/perioder"],
                 consumes = [APPLICATION_JSON_VALUE],
                 produces = [APPLICATION_JSON_VALUE])
 @Validated
 class PerioderOvergangsstønadController(private val perioderOvergangsstønadService: PerioderOvergangsstønadService) {
```

Comment:
Hvis det gjelder alle stønader ville jeg renamet; Stønadsperioder er kanskje greit? Isåfall kan det bygges ut med `/overgangsstønad` osv senere?